src/EA.Weee.DataAccess.Tests.Integration/EvidenceNote/NotesDataAccessIntegration.cs

```diff
@@ -64,6 +64,49 @@ public async Task GetAllNotes_ShouldMatchOnOrganisationId()
             }
         }
 
+        [Fact]
+        public async Task GetAllNotes_GivenSchemeQuery_ShouldOnlyRetrieveEvidenceNotes()
+        {
+            using (var database = new DatabaseWrapper())
+            {
+                var context = database.WeeeContext;
+                var dataAccess = new EvidenceDataAccess(database.WeeeContext, A.Fake<IUserContext>(), new GenericDataAccess(database.WeeeContext));
+
+                var organisation1 = ObligatedWeeeIntegrationCommon.CreateOrganisation();
+                var scheme = ObligatedWeeeIntegrationCommon.CreateScheme(organisation1);
+
+                context.Organisations.Add(organisation1);
+                
+                var aatf1 = ObligatedWeeeIntegrationCommon.CreateAatf(database, organisation1);
+
+                context.Aatfs.Add(aatf1);
+
+                await database.WeeeContext.SaveChangesAsync();
+
+                var note1 = NoteCommon.CreateNote(database, organisation1, scheme, aatf1);
+                
+                context.Notes.Add(note1);
+
+                var transferNote = NoteCommon.CreateTransferNote(database, organisation1, scheme);
```

Comment:
you need an evidence note created, not transfer right?
is not better to set up the note with await SetupSingleNote(context, database, NoteType.EvidenceNote);

Reply:
CreateNote creates a normal evidence note and CreateTransferNote creates the transfer note., they use their respective Note constructors. 

Then the assert just checks the the evidence note is retrieved.